### PR TITLE
Respond to GET requests with 200 OK

### PR DESCRIPTION
**Pull request by Zauberstuhl**
related to #45

### CODE REVIEW

File: lib/vines/stream/http.rb [hunk 1]
```diff
@@ -48,7 +50,7 @@ def valid_session?(sid)
       def process_request(request)
         if request.path == self.bind && request.options?
           request.reply_to_options
-        elsif request.path == self.bind
+        elsif request.path == self.bind && request.method != GET_METHOD
```
Comment (Zauberstuhl) [inline lib/vines/stream/http.rb]: I grep'ed through http://xmpp.org/extensions/xep-0124.html and was not able to find a use-case where BOSH is using `GET`. Correct me if I am wrong!

File: lib/vines/stream/http.rb [hunk 2]
```diff
@@ -58,7 +60,7 @@ def process_request(request)
           @session.request(request)
           @nodes.push(body)
         else
-          request.reply_with_file(self.root)
+          request.reply('It works!', 'text/html')
```
Comment (jhass) [inline lib/vines/stream/http.rb]: `text/plain` if you're not going to add any HTML ;)

File: lib/vines/stream/http.rb (hunk 1 above)
Comment (jhass) [inline lib/vines/stream/http.rb]: > Clients MUST send all HTTP requests as POST requests in any way permitted by RFC 1945 or RFC 2616.

Pretty clear.

Comment (Zauberstuhl) [inline lib/vines/stream/http.rb]: hehe :P cross-reading ftw

**Pull request merged**